relay-server/src/services/store.rs

```diff
@@ -1278,25 +1278,6 @@ struct UserReportKafkaMessage {
     event_id: EventId,
 }
 
-#[derive(Clone, Debug, Serialize)]
-struct SessionKafkaMessage {
```

Comment:
Was this unused?

Reply:
Yeah, not sure why it wasn't caught before.